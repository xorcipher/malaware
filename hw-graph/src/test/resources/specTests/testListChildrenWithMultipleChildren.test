# Tests that the getChildren method works properly for a graph in which the graph contains the specified node, and the node has multiple children.

# Return all the children of a given node within a given graph in a List.

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C

AddEdge graph A B edgeA
AddEdge graph A C edgeB

ListChildren graph A