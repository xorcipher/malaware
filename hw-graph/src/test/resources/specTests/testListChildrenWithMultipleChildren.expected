# Tests that the getChildren method works properly for a graph in which the graph contains the specified node, and the node has multiple children.

# Return all the children of a given node within a given graph in a List.

created graph graph

added node A to graph
added node B to graph
added node C to graph

added edge edgeA from A to B in graph
added edge edgeB from A to C in graph

the children of A in graph are: B(edgeA) C(edgeB)